docker/kind/agent-dev-build.yaml

```diff
@@ -1,113 +0,0 @@
----
```

Comment:
This was inlined into the k8s test module as rust code to allow it to be templated easily